src/index.html

```diff
@@ -8,5 +8,8 @@
 </head>
 <body>
   <app-root></app-root>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
```

Comment:
why are we hardcoding a script into the index here? This is an npm package and should be managed by npm. I thought we had bootstrap already installed, but maybe not. Get with @jordanDSmoothStack about this because I recall using some bootstrap stuff previously and I'm not sure if this will interfere with what he has already done. If it's not installed and we're importing it another way, we should ideally move to using npm to manage it.